pcweb/flexdown.py

```diff
@@ -1,25 +1,23 @@
 import flexdown
-from pcweb.styles.colors import c_color
-
 import reflex as rx
-from pcweb.templates.docpage import (
-    code_block_markdown,
-    code_block_markdown_dark,
-    code_comp,
-    docdemo,
-    docdemobox,
-    docgraphing,
-    doclink2,
-    h1_comp_xd,
-    h2_comp_xd,
-    h3_comp_xd,
-    h4_comp_xd,
-    text_comp,
-    list_comp,
-    definition,
-)
 
-from pcweb.styles.fonts import base, code
+from pcweb.styles.colors import c_color
+from pcweb.styles.fonts import base
+from pcweb.styles.fonts import code
+from pcweb.templates.docpage import code_block_markdown
```

Comment:
Can we keep the other style imports, this is quite verbose

Reply:
Sure, we can keep them together line.